src/common/cuepoint/cuepoint-manager.js

```diff
@@ -25,25 +19,27 @@ export class CuePointManager {
   }
 
   _createTextTrackCue(data: CuePoint): TextTrackCue {
```

Comment:
why are we not using the "private" indication here when methods are internal?

Reply:
what is "private" indication?